StereoKitC/xr_backends/openxr_view.cpp

```diff
@@ -704,12 +706,10 @@ bool openxr_render_frame() {
 
 	// Check each secondary display to see if it's active or not
 	for (int32_t i = 0; i < xr_displays_2nd.count; i++) {
-		if (xr_displays_2nd[i].active != (bool32_t)xr_display_2nd_states[i].active) {
-			xr_displays_2nd[i].active  = (bool32_t)xr_display_2nd_states[i].active;
+		xr_displays_2nd[i].active  = (bool32_t)xr_display_2nd_states[i].active;
 
-			if (xr_displays_2nd[i].active) {
-				openxr_display_swapchain_update(&xr_displays_2nd[i]);
-			}
+		if (xr_displays_2nd[i].active) {
+			openxr_display_swapchain_update(&xr_displays_2nd[i]);
```

Comment:
Was this chunk a critical part of the fix? Since `openxr_display_swapchain_update` does also have some checks to prevent redundant updates, this is probably okay. However, [the spec](https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#_secondary_view_configuration_properties) does kinda indicate that the `xrEnumerateViewConfigurationViews` call in `openxr_display_swapchain_update` should only really be necessary when `active` changes.

Reply:
This is only critical if the secondary display is active prior to SK running. If we set the display as initially inactive (same as `XrSecondaryViewConfigurationStateMSFT`), then we can keep this check! That works too :)